src/mrd/tests/unit/test_generate_mrd_report.py

```diff
@@ -39,3 +40,11 @@ def test_generate_mrd_report(output_path, resources_dir):
 
     # assert report_html is not empty
     assert output_report_html.stat().st_size > 0
+
+    # assert h5 output values are as expected
+    h5_output = str(output_path / "test_report.tumor_fraction.h5")
+    h5_expected = str(resources_dir / "test_report.tumor_fraction.expected_output.h5")
+    with pd.HDFStore(h5_expected) as store:
+        h5_keys = store.keys()
```

Comment:
Please make sure the key are the same in both cases

```suggestion
with pd.HDFStore(h5_expected) as store:
    h5_keys = store.keys()
with pd.HDFStore(h5_output) as store:
    h5_keys_output = store.keys()
assert h5_keys ==  h5_keys_output 
```